recipes/photospline/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "photospline" %}
+{% set org = "cnweaver" %}
+{% set version = "2.0.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "0a3368205a7971a6919483ad5b5f0fbebb74614ec1891c95bb6a4fc9d3b950d4" %}
```

Comment:
Please reduce the excess of jinja to just the version. There is no need to create variables for things that will be used only once and never updated.